### DISCUSSION TITLE
Storage Expansion on Manager Node

### DISCUSSION THREAD

**Discussion by ow111** [Security-Onion-Solutions/securityonion#2610] [Unsupported Versions] [answered]
Running SOv2.3.21 , the prepackaged VM installed from ISO

when running so-features-enable the VM runs out of disk space. I can expand the disk in the hypervisor but i will need to expand the logical volume in the OS, is there a certain way you folks recommend to do this without breaking the backend?

**Comment by weslambert** [comment, marked as answer]
You can add more storage to the volume group via LVM.

**Comment by ow111** [comment]
Thanks wes I was able to expand the storage, now getting a GPG key error 


`gpg: Signature made Mon 21 Dec 2020 10:59:34 PM UTC using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>"  `

I tried downloading the "KEYS" file from your github,  however when trying to import with gpg --import </path/to/KEYSfile> I get the error 
```
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
```
